packages/lodestar-cli/src/cmds/dev/options.ts

```diff
@@ -1,16 +1,45 @@
 import {Options} from "yargs";
 import {IGlobalArgs} from "../../options";
 import {beaconOptions, IBeaconOptions} from "../beacon/options";
+import defaultOptions from "@chainsafe/lodestar/lib/node/options";
 
 export const devRunOptions = {
   ...beaconOptions,
 
   "sync.minPeers": {
     type: "number",
-    default: 2,
+    default: 0,
     group: "sync",
   } as Options,
 
+  "network.maxPeers": {
```

Comment:
Aren't the options `"network.maxPeers"`, `"eth1.enabled"`, and `"api.rest.enabled"` included in `beaconOptions`?

Reply:
yes, but wrong defaults for dev chain. In dev command we don't wanna connect to other peers, but we do wanna get out of syncing (min peers) and have api enabled by default (as it's used by validator)